index.html

```diff
@@ -0,0 +1,114 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-1BmE4kWBq78iYhFldvKuhfTAU6auU8tT94WrHftjDbrCEXSU1oBoqyl2QvZ6jIW3" crossorigin="anonymous">
+    <link rel="stylesheet" href="css/index.css">
+    <script src="js/index.js" type="module" async></script>
+    <title>Creative Commons Global Summit 2015</title>
+  </head>
+  <body>
+    <main id="main" class="container-fullwidth">
+      <header>
+        <nav id="nav" class="container-fullwidth navbar">
+          <div class="desktop-visible" id="top-nav" >
+            <a href="https://facebook.com"><img src="assets/icons/facebook.png" alt="Facebook logo"></a>
+            <a href="https://twitter.com"><img src="assets/icons/twitter.png" alt="Twitter logo"></a>
+            <a href="#">Logout</a>
+            <a href="#">Mypage</a>
+            <a href="#">English</a>
+          </div>
+          <img src="assets/icons/mobile-menu.png" alt="Menu hamburger button" id="hamburger" class="nav-item mobile-visible">
+          <div class="nav-options row-fullwidth mobile-visible" id="bottom-nav">
+            <a class="desktop-visible" id="logo" href="index.html"><img src="assets/images/logo-clickable.png" alt="Clickable logo"></a>
+            <img class="mobile-visible close-menu" src="assets/icons/close-icon.png" alt="Close menu button.">
+            <div class="mobile-menu">
+              <a class="nav-option current-page" href="html/about.html">About</a>
+              <a class="nav-option" href="#programs">Program</a>
+              <a class="nav-option mobile-visible" href="#join-summit">Join</a>
+              <a class="nav-option desktop-visible" href="#seewhole">Join</a>
+              <a class="nav-option" href="#sponsors">Sponsor</a>
+              <a class="nav-option" href="https://creativecommons.org/tag/news/">News</a>
+              <a class="nav-option" href="https://creativecommons.org/campaign/">CC Campaign</a>
```

Comment:
- [ ] Please wrap these links in a list, You can use `<ul>.` When we similar elements on the page It is a good practice to wrap them in the list and it will also make sure the code is semantic.